### PR TITLE
feat: migrated app to Quasar

### PR DESCRIPTION
**Pull request by impelcrypto**
* Migrated the app to Quasar framework
* Added Polkadot.js libraries
* Added wallet connect
* Fetch the information for staked dApps by connected address

To-do:
* Making network configurable (default: Shiden)
* Modifying mint logic once contract has been finalized

Screenshot 2022-03-08 at 7 13 59 PM

image

image

image

### CODE REVIEW

File: src/boot/api.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        href: favicon,
+      },
+    },
+    meta: opengraphMeta,
+  });
+
+  const { api, extensions } = await connectApi(endpoint, store);
+  $api.value = api;
+
+  const result = await api.query.dappsStaking.registeredDapps({
```
Comment (hoonsubin) [inline src/boot/api.ts:43]: What is this function for? I don't think it's used anywhere

Comment (impelcrypto) [reply src/boot/api.ts]: Yes, you are right. I've removed it.

File: src/components/Mint/EthereumWallet.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    };
+
+    const disconnect = () => {
+      store.dispatch('general/disconnect');
+    };
+
+    const mint = () => {
+      const contract = mintContract.value as Contract;
+      contract?.methods
+        .mint('1')
```
Comment (hoonsubin) [inline src/components/Mint/EthereumWallet.vue:46]: what about making this configurable through a separate file? Including the contract address

Comment (impelcrypto) [reply src/components/Mint/EthereumWallet.vue:46]: Sure, I'll move the minting logic into a hook.

Comment (impelcrypto) [reply src/components/Mint/EthereumWallet.vue]: Fixed in [this](https://github.com/AstarNetwork/astarbase/pull/5/commits/ed34d9a5470f0836d8e92128f9044cdec8736953) commit

**Review by hoonsubin — APPROVED** [review]
LGTM
The actual minting implementation and the AstarBase implementation will come later

**Pull request merged**